README.md

```diff
@@ -45,160 +88,88 @@ To run a specific unittest:
 
     docker-compose run --use-aliases boulder go test ./ra
 
-The configuration in docker-compose.yml mounts your
-[`$GOPATH`](https://golang.org/doc/code.html#GOPATH) on top of its own
-`$GOPATH`. So you can edit code on your host and it will be immediately
-reflected inside Docker images run with docker-compose.
+The configuration in docker-compose.yml mounts your `$GOPATH` on top of its own `$GOPATH` so you can edit code on your host and it will be immediately reflected inside the Docker containers run with docker-compose.
 
-If docker-compose fails with an error message like "Cannot start service
-boulder: oci runtime error: no such file or directory" or "Cannot create
-container for service boulder" you should double check that your `$GOPATH`
-exists and doesn't contain any characters other than letters, numbers, `-`
-and `_`.
+If docker-compose fails with an error message like "Cannot start service boulder: oci runtime error: no such file or directory" or "Cannot create container for service boulder" you should double check that your `$GOPATH` exists and doesn't contain any characters other than letters, numbers, `-` and `_`, and that it doesn't contain any dangling symlinks.
 
-If you have problems with Docker, you may want to try [removing all containers
-and volumes](https://www.digitalocean.com/community/tutorials/how-to-remove-docker-images-containers-and-volumes).
+If you have problems with Docker, you may want to try [removing all containers and volumes](https://www.digitalocean.com/community/tutorials/how-to-remove-docker-images-containers-and-volumes).
 
-By default, Boulder uses a fake DNS resolver that resolves all hostnames to
-127.0.0.1. This is suitable for running integration tests inside the Docker
-container. If you want Boulder to be able to communicate with a client running
-on your host instead, you should find your host's Docker IP with:
+By default, Boulder uses a fake DNS resolver that resolves all hostnames to 127.0.0.1. This is suitable for running integration tests inside the Docker container. If you want Boulder to be able to communicate with a client running on your host instead, you should find your host's Docker IP with:
 
     ifconfig docker0 | grep "inet addr:" | cut -d: -f2 | awk '{ print $1}'
 
-And edit docker-compose.yml to change the FAKE_DNS environment variable to
-match.
+And edit docker-compose.yml to change the FAKE_DNS environment variable to match. This will cause Boulder to use the local system resolver available on your host if one is available.
```

Comment:
The second sentence is inaccurate. Instead I would say "This will cause Boulder's stubbed-out DNS resolver (sd-test-srv) to respond to all A queries to the address in FAKE_DNS". This is a common source of confusion for just about everyone, myself included. :-D It might be nice to do this in a more intuitive way in the future.